{
  "file_data": [
    {
      "desc": {
        "cluster": true,
        "cust_a": {
          "val": 17
        },
        "n_out": 4
      },
      "geom": {
        "p": [
          "{droG`pfcK"
        ]
      },
      "id": "6-2",
      "title": "Area Outage"
    },
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "val": 13
        },
        "etr": "2020-12-02T12:45:00-0400",
        "start": "2020-12-02T03:18:17-0400"
      },
      "geom": {
        "p": [
          "qceqGlnq|J"
        ]
      },
      "id": "8",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
